composer.json

```diff
@@ -36,6 +36,7 @@
         "phpstan/phpstan": "^1.10.25",
         "predis/predis": "^2.2",
         "rector/rector": "^0.16.0",
+        "symfony/cache": "^7.0",
```

Comment:
is this really mandatory?

Reply:
I strongly agree on having as few dependencies as possible, but without that library I get this error while running `DoctrineVectorStoreTest`
```
RuntimeException: The Doctrine setup tool cannot configure caches without symfony/cache. Please add symfony/cache as explicit dependency or pass your own cache implementation.
at vendor/doctrine/orm/src/ORMSetup.php:190
at vendor/doctrine/orm/src/ORMSetup.php:166
at vendor/doctrine/orm/src/ORMSetup.php:107
at tests/Integration/Embeddings/VectorStores/Doctrine/DoctrineVectorStoreTest.php:17
```

Reply:
Do you have in mind something like this?
https://github.com/theodo-group/LLPhant/pull/130/commits/2f8a82d5e9df76ce238388e96e93d1b1b3107ba2

Reply:
Do you have in mind something like this?
https://github.com/theodo-group/LLPhant/pull/130/commits/2f8a82d5e9df76ce238388e96e93d1b1b3107ba2